python/llm/src/bigdl/llm/transformers/models/baichuan2.py

```diff
@@ -50,24 +50,25 @@ def baichuan_13b_rms_norm_forward(self, hidden_states):
     if hidden_states.device.type == "xpu" and not (self.training and hidden_states.requires_grad):
         if get_ipex_version() <= "2.0.110+xpu":
             import linear_q4_0
-            hidden_states = linear_q4_0.fused_rms_norm(hidden_states,
+            result = linear_q4_0.fused_rms_norm(hidden_states,
                                                        [self.weight.size(0)],
                                                        self.weight,
                                                        None,
                                                        self.epsilon)
+            # if nelement == 0, means fused norm failed, go back to python implement.
+            if result.nelement != 0:
```

Comment:
do you need to check for `2.1`?

Reply:
No, if can not be fused, fast_rms_norm will go to another native implement. We don't have the native implement, so we go back to python.

Reply:
You should use fused_rms_norm for both 2.0 & 2.1